functions.js

```diff
@@ -0,0 +1,16 @@
+function openCloseMenu() {
+  const x = document.querySelector('.mobile-menu');
+  const y = document.querySelector('body');
+  if (x.style.display === 'none') {
+    x.style.display = 'flex';
+    y.style.overflow = 'hidden';
+  } else {
+    x.style.display = 'none';
+    y.style.overflow = 'scroll';
+  }
+}
+document.getElementById('onclick-open').addEventListener('click', openCloseMenu);
+document.getElementById('onclick-close').addEventListener('click', openCloseMenu);
+document.getElementById('onclick-portfolio').addEventListener('click', openCloseMenu);
+document.getElementById('onclick-about').addEventListener('click', openCloseMenu);
+document.getElementById('onclick-contact').addEventListener('click', openCloseMenu);
```

Comment:
## WELL DONE DEV 😀👏

- [ ] [OPTIONAL] This is my optional suggestion which means not required but good to fix, ok 😉? Please make this code dry that helps you and also others like me to read easily and to understand. **_TIP_**: Store ids in array and loope over them using `forEach`. 

![image](https://user-images.githubusercontent.com/69011963/150823974-4038c9fb-c620-4bd2-a333-1644a88469aa.png)

Reply:
Thanks, I did it.